starttls-policy/starttls_policy/configure.py

```diff
@@ -49,10 +55,25 @@ def manual_instructions(self):
                     mta_name=self.mta_name,
                     instructions=self._instruct_string()))
 
+    def expired_warning(self):
```

Comment:
Prefix function with an underscore to indicate it's meant for internal use only

Reply:
I noticed `manual_instructions` method is named as public so I decided to do same with `expired_warning` in order to provide same "visibility" for somewhat similar methods. Is it wrong?

Reply:
Great question!
From a high level, we wanted any consumer of a ConfigGenerator to be able to do two things:
 - Generate a configuration file for an MTA
 - Install that configuration file to that MTA

Which correspond to the `generate` and `manual_instructions`, respectively. The `manual_instructions` name is a bit of a misnomer; it was originally `install`, but was changed since we were concerned about modifying people's MTA configs.

So although `manual_instrcutions` and `expired_warning` both do similar things under the hood (writing to a std fd), their intended use is different-- a consume of ConfigGenerator should ideally never have to bring up an expiry warning since we expect `generate` to do that.

Reply:
Ok, got it! Fixed.